src/store/todo/todoSagas.ts

```diff
@@ -21,8 +29,62 @@ function* getTodoSagas(action: todosTypes.FetchTodos) {
   }
 }
 
+function* updateTodoTask(action: todosTypes.UpdateTodo) {
+  const { payload } = action;
+  try {
+    yield put(todosActions.updateTodoPending());
+
+    const todos: TodoType[] = yield select((state) => state.todosReducer.todos);
+
```

Comment:
Like I siad, use your delay method here to simulate asynch, then user will be able to see loading indicator.